test/test_tensorexpr_pybind.py

```diff
@@ -0,0 +1,40 @@
+import torch
+import unittest
+
+from torch.testing._internal.jit_utils import JitTestCase
+
+class kernel_arena_scope(object):
+    def __enter__(self):
+        self.scope = torch._C.te.KernelScope()
+
+    def __exit__(self, typ, val, traceback):
+        self.scope = None
+
+class TestTensorExprPyBind(JitTestCase):
+    def test_simple_sum(self):
+        with kernel_arena_scope():
```

Comment:
Not a big deal for this PR but omg I want to get rid of KernelArena before we start advertising this API.  Weird low-level memory management hacks shouldn't be leaking up into Python :-p

Reply:
100% agree 😃 
I think even if we keep the arena for now, there is probably a more elegant way of using it for python bindings than I did here.